### ISSUE TITLE
500 error on request containing &amp;

### ISSUE THREAD

**Issue by bwm-epimorphics** [epimorphics/elda#161]
The following

  http://lr-pres-staging-b.epimorphics.net/def/ppi.html?_page=0&amp;_sort=-topConceptOf.comment

gets a 500 error response.

Replace the &amp; with a simple apersand and it works fine.

**Comment by ehedgehog** [comment]
What generates that URL?

Chris

On 10 December 2015 at 13:26, bwm-epimorphics <notifications@github.com>
wrote:

> Assigned #161 <https://github.com/epimorphics/elda/issues/161> to
> @ehedgehog <https://github.com/ehedgehog>.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/epimorphics/elda/issues/161#event-488159906>.
>



-- 
Chris "allusive" Dollin

**Comment by bwm-epimorphics** [comment]
I have no idea how the request is generated.

I'm seeing 500 errors in the log files.  The request is coming from an IP address I don't recognise.

Brian

**Comment by ehedgehog** [comment]
Looks like then that there are two problems.

One is that /something/ is generating naff queries. We don't know
what -- it could be a generated link on the html page, or it could be
some other app misconstructing a URL. If it's our html page it
would be nice to find and fix it.

The other is that should be a 400 error rather than a 500. Fixing
that probably requires finding out exactly what the error 500
is. It's possible it has been fixed since .14 since we've rearranged
error pages.

Any chance of a log with DEBUG entries in it?

Chris


On 10 December 2015 at 13:56, bwm-epimorphics <notifications@github.com>
wrote:

> I have no idea how the request is generated.
>
> I'm seeing 500 errors in the log files. The request is coming from an IP
> address I don't recognise.
>
> Brian
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/epimorphics/elda/issues/161#issuecomment-163625816>.
>



-- 
Chris "allusive" Dollin
